docs/documentation/gui.md

```diff
@@ -9,7 +9,7 @@ The Dimlpfidex Graphical User Interface was made to help with **`JSON configurat
 2. Follow the next section to learn how to use the GUI, a tool made to ease your experience with **`JSON configuration files`**.
 
 ## Description
-[JSON](https://www.json.org/json-en.html) or `JavaScript Object Notation` is a lightweight data interchange format that is easy for humans to read, and easy for machines to parse and generate. It is used to structure data in a way that is both human-readable and machine-friendly, making it an ideal choice for configuration files.
+[JSON](https://www.json.org/json-en.html) or `JavaScript Object Notation` is a lightweight data-interchange format that is easy for humans to read, and easy for machines to parse and generate. It is used to structure data in a way that is both human-readable and machine-friendly, making it an ideal choice for configuration files.
```

Comment:
Aim for conciseness, unless it is about the algorithms. I'd reformulate the sentence into "The configuration files use teh JSOn format".